webhook/resourcesemantics/defaulting/defaulting.go

```diff
@@ -186,6 +186,9 @@ func (ac *reconciler) reconcileMutatingWebhook(ctx context.Context, caCert []byt
 			MatchExpressions: []metav1.LabelSelectorRequirement{{
 				Key:      "webhooks.knative.dev/exclude",
 				Operator: metav1.LabelSelectorOpDoesNotExist,
+			},{
```

Comment:
Can we add a comment about where the `control-plane` annotation comes from (AKS), just for posterity?

Reply:
Maybe link this upstream issue as well
https://github.com/Azure/AKS/issues/1771